packages/mui-docs/src/i18n/i18n.tsx

```diff
@@ -81,6 +81,16 @@ export function useSetUserLanguage() {
 
 const warnedOnce: Record<string, boolean> = {};
 
+const warn = (userLanguage: string, key: string, ignoreWarning: boolean) => {
```

Comment:
function for top-level declarations

```suggestion
function warn(userLanguage: string, key: string, ignoreWarning: boolean) {
```

Reply:
We should the `warnOnce` helper in the future https://github.com/mui/mui-x/pull/13911.